UPGRADING.md

```diff
@@ -39,6 +39,48 @@ app.intent("tellme", (request, response) => {
 });
 ```
 
+#### Changes to ExpressJS Support
+
+After changes, `alexa.express()` no longer requires both `expressApp` and `router` to be passes in. Instead, just pass in either an express app or router. And alexa-app would be bind to it.
```

Comment:
That's not the whole change. Also it's unclear what `alexa` is, it should be `app`, and "to be passed in" is redundant. And the next sentence says the same thing. Maybe:

```
Express.js integration via `app.express()` now requires one of `expresApp` or `router`. When using a router, it will no longer be automatically mounted inside the express app and you may need to implement that outside of this call.
```